hack/release.toml

```diff
@@ -1,9 +1,9 @@
 # commit to be tagged for the new release
 commit = "HEAD"
 
-project_name = "omni-cloud-provider-qemu"
-github_repo = "siderolabs/omni-cloud-provider-qemu"
-match_deps = "^github.com/(siderolabs/[a-zA-Z0-9-]+)$"
+project_name = ""
```

Comment:
weird, this shoudn't be changed

Reply:
reverted that manually, they stay after rekres.